hummingbird/ml/operator_converters/linear_classifier.py

```diff
@@ -0,0 +1,98 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import torch
+import numpy as np
+from onnxconverter_common.registration import register_converter
+
+"""
+Converters for scikit-learn linear models: LinearRegression, LogisticRegression, LinearSVC, SGDClassifier, LogisticRegressionCV
+"""
+
+
+class SklearnLinearModel(torch.nn.Module):
+    def __init__(self, coefficients, intercepts, classes, multi_class, device, is_linear_regression=False):
+        super(SklearnLinearModel, self).__init__()
+        self.coefficients = torch.nn.Parameter(torch.from_numpy(coefficients), requires_grad=False)
+        self.intercepts = torch.nn.Parameter(torch.from_numpy(intercepts), requires_grad=False)
+        self.classes = torch.nn.Parameter(torch.IntTensor(classes), requires_grad=False)
+        self.multi_class = multi_class
+        self.regression = is_linear_regression
+
+        self.perform_class_select = False
+        if min(classes) != 0 or max(classes) != len(classes) - 1:
+            self.perform_class_select = True
+
+        self.binary_classification = False
+        if len(classes) == 2:
+            self.binary_classification = True
+
+    def forward(self, x):
+        if self.multi_class == "multinomial":
```

Comment:
Can we pull `torch.addmm(self.intercepts, x, self.coefficients)` out put it in the first line? Then `if self.regression` will just return output.

Reply:
@interesaaat  - ok, please take a look and make sure this didn't break anything.

Mmmmm paper deadline code from December. delicious.